homeassistant/components/smappee/config_flow.py

```diff
@@ -21,3 +25,124 @@ class SmappeeFlowHandler(
     def logger(self) -> logging.Logger:
         """Return logger."""
         return logging.getLogger(__name__)
+
+    async def async_step_zeroconf(self, discovery_info):
+        """Handle zeroconf discovery."""
+
```

Comment:
See comment below, from our point of view there is no need to abort.

Reply:
Aborting the local flow with a conflicting existing cloud instance is clear to me (because the cloud instance has been loaded completely), but I don't really see how to implement the other way around (aborting the cloud instance; we need the list of serial numbers which implies a complete setup).

Reply:
Resolved by only allowing (mutiple) LOCAL or CLOUD but not both.